src/SUMMARY.md

```diff
@@ -20,7 +20,7 @@
 - [The compiler testing framework](./tests/intro.md)
     - [Running tests](./tests/running.md)
     - [Adding new tests](./tests/adding.md)
-    - [Using `compiletest` + commands to control test execution](./compiletest.md)
+    - [Using `compiletest` commands to control test execution](./compiletest.md)
```

Comment:
Should we change the h1 header of `src/compiletest.md` as well? The current wording is `# compiletest` and it's unclear a bit.